pkg/cloudprovider/launchtemplate.go

```diff
@@ -314,7 +316,7 @@ func (p *LaunchTemplateProvider) kubeServerVersion(ctx context.Context) (string,
 	version := fmt.Sprintf("%s.%s", serverVersion.Major, strings.TrimSuffix(serverVersion.Minor, "+"))
 	p.cache.SetDefault(kubernetesVersionCacheKey, version)
 	if p.cm.HasChanged("kubernete-version", version) {
-		logging.FromContext(ctx).Debugf("Discovered kubernetes version %s", version)
+		logging.FromContext(ctx).With("version", version).Debugf("discovered kubernetes version")
```

Comment:
This might be confused with the Karpenter controller version, I'd go with `k8s-version`

Reply:
`kubernetes-version`